examples/archive.html

```diff
@@ -137,6 +145,8 @@ <h3 id="offcanvasRightLabel">Images</h3>
         });
     });
 
+    saveButton.addEventListener('click',generateExportJson(imageGroup))
```

Comment:
Very close here! I think we will want to run it like this:

```suggestion
    saveButton.addEventListener('click', function() {
      alert(placedImagesCollection.generateExportJson()); // we'll do something better than alert in the next PR
    }
```